near-fetch/src/query.rs

```diff
@@ -591,14 +592,28 @@ impl Client {
         sender_id: &AccountId,
         tx_hash: CryptoHash,
     ) -> Result<FinalExecutionOutcomeView> {
-        self.rpc_client
+        let response = self
+            .rpc_client
             .call(methods::tx::RpcTransactionStatusRequest {
                 transaction_info: methods::tx::TransactionInfo::TransactionId {
-                    account_id: sender_id.clone(),
-                    hash: tx_hash,
+                    sender_account_id: sender_id.clone(),
+                    tx_hash,
                 },
+                wait_until: TxExecutionStatus::Executed,
```

Comment:
Is this the right wait_until status to use?

Reply:
The `wait_until` should be an optional param to pass through with Executed as the default

Reply:
This will require that you match on multiple return types below, since for `INCLUDED` and maybe `INCLUDED_FINAL` the txn hash is returned.

Reply:
I think you also might need to update the rpc call in other parts of the code since this only queries the transaction status. We'll also need to modify it for actually sending the txn

Reply:
>We'll also need to modify it for actually sending the txn

You're right - the compiler only complained about this function. Turns out, the `send_tx` methods are using the old `broadcast_tx` methods on the RPC.

Reply:
The updated code should be based on https://github.com/near/near-jsonrpc-client-rs/blob/master/examples/send_tx.rs#L62